synctl

```diff
@@ -71,7 +71,19 @@ def abort(message, colour=RED, stream=sys.stderr):
     sys.exit(1)
 
 
-def start(configfile, daemonize=True):
+def start(configfile: str, daemonize=True) -> bool:
+    """Attempts to start synapse.
+    Args:
+        configfile: path to a yaml synapse config file
+        daemonize: whether to daemonize synapse or keep it attached to the current
+            session
+
+    Returns:
+        boolean stating whether synapse started successfully if daemonize is True
+
+        doesn't return unless interupted if daemonize is false.
```

Comment:
this isn't quite true. It will return as soon as synapse exits - which may be very soon if there is an error starting synapse - or may be hours or days later.